### DISCUSSION TITLE
raylib uwp

### DISCUSSION THREAD

**Discussion by kororro** [raysan5/raylib#1646] [General]
Hi.
I have tested raylib UWP, it works but can I compile it to windows desktop?
I mean I can run UWP test in VS2019 community IDE, but when I try run that .exe file from file explorer, it requires some odd .dll file (I cant check it now, atm I have not installed uwp, I have reistalled windows and stuff).

It is possible?
Why? Well when I played with ANGLE, I noticed that even emulated D3D11 is faster than straight OpenGL.
Im not going to do ANGLE support to raylib, Ray wrote that it is not worth it  (I tried, was too hard, didnt succeeded at all   (maybe someday I try again)).

Btw I remembered there was angle project under projects/ dir, but not any more, dont remember when it has been removed (or was it on PR or issues, cant remember). Well, not needed, original question above is more important to me.

-m

**Comment by raysan5** [comment]
Actually, I don't know. I think it should work but I did not tested it much beside just compiling the example. raylib UWP uses ANGLE by default.

**Comment by raysan5** [comment]
I'm afraid raylib UWP support has been removed in latest `raylib 4.0`.
